Makefile

```diff
@@ -0,0 +1,34 @@
+gen-api: mkdir gen-curp-api gen-xline-api
+
+mkdir:
+	mkdir api && \
+	cd api && \
+	mkdir xline && \
+	mkdir curp
+
+gen-xline-api:
+	cd api/xline/ && \
+	rm -rf * && \
+	cd ../../xline-proto/src/ && \
+	protoc \
+		--experimental_allow_proto3_optional \
+		--go_out=../../api/xline \
+		--go_opt=paths=source_relative \
+    	--go-grpc_out=../../api/xline \
```

Comment:
Please use space instead of Tab here for consistency

Reply:
done

Reply:
Makefile doesn't seem to support the use of spaces for indentation